packages/taco/integration-test/sign.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Domain, getPorterUris } from '@nucypher/shared';
+import { ethers } from 'ethers';
+import { beforeAll, describe, expect, test } from 'vitest';
+
+import { initialize } from '../src';
+import { signUserOp } from '../src/sign';
+import { UserOperation } from '../src/types';
+
+const RPC_PROVIDER_URL = 'https://rpc-amoy.polygon.technology';
+const SIGNER_PRIVATE_KEY = '0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86';
+const DOMAIN = 'lynx';
+const RITUAL_ID = 27;
+const CHAIN_ID = 80002;
```

Comment:
This test won't work with these values:
- the signing coordinator is on `sepolia` and not `amoy` so the provider needs to be for sepolia.
- `27` is not a valid cohort id for signing, `2` should be used
- cohort id `2` is not configured on amoy (`80002`), but it is for sepolia (`11155111`)